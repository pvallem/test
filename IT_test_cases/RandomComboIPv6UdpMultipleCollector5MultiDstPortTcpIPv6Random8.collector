bidir_flow_info {
  key {
    src_address: "\215fwK\354\367\264\262\372\000N\3678\303lC"
    dst_address: "\247g5O\257\3104\340\235\345\237,V\330v\030"
    src_port: 4928
    dst_port: 57296
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV6
  }
  reverse_flow_info {
    start_time: 1388563200000010
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000010
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\001\000\000\000\000\000\000\000\000\000\000\000\000\000\000\001"
    dst_address: "\001\000\000\000\000\000\000\000\000\000\000\000\000\000\000d"
    src_port: 12345
    dst_port: 12346
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV6
  }
  flow_info {
    start_time: 1388563200000020
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000020
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\264\022h@\223`\026K\237K\200!,\370\321\236"
    dst_address: "\343A\254\3400Z\274tKi\377\270\004.U\336"
    src_port: 50712
    dst_port: 54393
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV6
  }
  reverse_flow_info {
    start_time: 1388563200000030
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000030
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\001\000\000\000\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\001\000\000\000\000\000\000\000\000\000\000\000\000\000\000d"
    src_port: 12345
    dst_port: 12345
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV6
  }
  flow_info {
    start_time: 1388563200000000
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000000
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\001\000\000\000\000\000\000\000\000\000\000\000\000\000\000\002"
    dst_address: "\001\000\000\000\000\000\000\000\000\000\000\000\000\000\000d"
    src_port: 12345
    dst_port: 12347
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV6
  }
  flow_info {
    start_time: 1388563200000040
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000040
    sensor_id: "127.0.0.1:5675"
  }
}
