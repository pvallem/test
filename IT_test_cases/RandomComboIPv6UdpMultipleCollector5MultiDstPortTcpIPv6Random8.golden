flow_info {
  key {
    src_address: "\001\000\000\000\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\001\000\000\000\000\000\000\000\000\000\000\000\000\000\000d"
    src_port: 12345
    dst_port: 12345
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV6
  }
  start_time: 1388563200000000
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000000
}
flow_info {
  key {
    src_address: "\247g5O\257\3104\340\235\345\237,V\330v\030"
    dst_address: "\215fwK\354\367\264\262\372\000N\3678\303lC"
    src_port: 57296
    dst_port: 4928
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV6
  }
  start_time: 1388563200000010
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000010
}
flow_info {
  key {
    src_address: "\001\000\000\000\000\000\000\000\000\000\000\000\000\000\000\001"
    dst_address: "\001\000\000\000\000\000\000\000\000\000\000\000\000\000\000d"
    src_port: 12345
    dst_port: 12346
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV6
  }
  start_time: 1388563200000020
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000020
}
flow_info {
  key {
    src_address: "\343A\254\3400Z\274tKi\377\270\004.U\336"
    dst_address: "\264\022h@\223`\026K\237K\200!,\370\321\236"
    src_port: 54393
    dst_port: 50712
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV6
  }
  start_time: 1388563200000030
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000030
}
flow_info {
  key {
    src_address: "\001\000\000\000\000\000\000\000\000\000\000\000\000\000\000\002"
    dst_address: "\001\000\000\000\000\000\000\000\000\000\000\000\000\000\000d"
    src_port: 12345
    dst_port: 12347
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV6
  }
  start_time: 1388563200000040
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000040
}
